java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -19,24 +19,25 @@
 package org.apache.arrow.vector;
 
 import io.netty.buffer.ArrowBuf;
+import org.apache.arrow.memory.BufferAllocator;
 
-class BitVectorHelper {
+public class BitVectorHelper {
```

Comment:
I think we made this package private previously. Is this used now outside the package?

Reply:
I think the functions in this class needs some doc.

Reply:
Yes, it is used in Json specific code as well and in some tests too.

Reply:
I see. In that case, maybe add doc in this class that this is internally API?